theme/snippets/add-to-cart-container.liquid

```diff
@@ -2,32 +2,45 @@
 {% capture button_label %}{{"snippets.add_to_cart_container.add_to_cart_btn.text" | t }}{% endcapture %}
 {% assign button_disabled = false %}
 {% if product.available %}
-  {% if product_in_cart %}
-    {% capture button_label %}{{"snippets.add_to_cart_container.added_to_cart_btn.text" | t }}{% endcapture %}
-    {% assign button_disabled = true %}
-  {% endif %}
-{% else %}
   {% case product_type %}
     {% when "Course" %}
     {% when "Event" %}
       {% capture button_label %}{{"snippets.add_to_cart_container.registration_closed_btn.text" | t }}{% endcapture %}
       {% assign button_disabled = true %}
+  {% endcase %}
     {% else %}
       {% capture button_label %}{{"snippets.add_to_cart_container.sold_out_btn.text" | t }}{% endcapture %}
       {% assign button_disabled = true %}
-  {% endcase %}
 {% endif %}
 
 <div class="AddToCartContainer">
   <form data-add-to-cart-form data-product-type={{product_type}} action="/cart/add" method="post" enctype="multipart/form-data" id="AddToCartForm">
-    <select class="{% if product.variants.size <= 1 %} none {% endif %}" name="variant" data-variant-selector>
-    {% for variant in product.variants %}
-      <option value="{{ variant.id }}">
-        {{ variant.title }}
-      </option>
-    {% endfor %}
-  </select>
-    <button data-variant-id={{product.first_available_variant.id}} data-add-to-cart-button {% if button_disabled %} disabled {% endif %} class="w100 color-white uppercase sans-light-sm radius-xs bg-color-black py_75" type="submit" name="add" id="AddToCart">
+    <div class="AddToCartContainer__quantity-select-wrapper overflow-hidden relative {% if product.variants.size <= 1 %} mb1_75 {% endif %}">
+      <label for="quantity" type="number" name="quantity" aria-label={{"snippets.add_to_cart_container.quantity_spinner.label" | t }}></label>
+      <select class="sans-xs uppercase radius-xs border-black w100 pl1_125 py_5" name="quantity" id="Quantity" data-quantity-selector>
+        <option value="" selected="selected">{{"snippets.add_to_cart_container.quantity_spinner.label" | t }}</option>
+        {% comment %}
+        TO-DO: Add notification if user adds quantity > product inventory
+        {% endcomment %}
+        {% for i in (1...10) %}
+          <option value="{{ i }}">{{ i }}</option>
+        {% endfor %}       
```

Comment:
Is there a reason why the quantity is a select from 1-10 and not a number input?

Reply:
Gooood question! It was originally a number input but I changed it to a `<select>` because the design is a dropdown menu! When I looked into it, it seems like the way to create that design is a `<select>`. What do you think?

Reply:
If that's what is designed, then yeah a dropdown makes sense! The only issue I can imagine is if there is someone who wants to add more than 10... but I don't think this will come up and they can just refresh and add in multiple batches i spose